recipes-bsp/u-boot/files/ae350-ax45mp/0002-mmc-ftsdc010_mci-Support-DTS-of-ftsdc010-driver-for-.patch

```diff
@@ -0,0 +1,34 @@
+From 16aad5594e08550295ea3c12c1c9ed6f64774748 Mon Sep 17 00:00:00 2001
+From: Rick Chen <rick@andestech.com>
+Date: Tue, 29 Mar 2022 13:41:10 +0800
+Subject: [PATCH] mmc: ftsdc010_mci: Support DTS of ftsdc010 driver for
+ generic dma
+
+The ftsdc010 driver has been implemented for generic dma in Linux
+kernel. And its compatible is andestech,atfsdc010g to distinguish
+the legacy andestech,atfsdc010 which is not for generic dma.
+
+Althought the ftsdc010_mci driver in U-Boot does not use dma, but
+it still can work well with the mmc node for generic dma. So add
+the compatible string to support it.
+
+Signed-off-by: Rick Chen <rick@andestech.com>
```

Comment:
Can you add the upstream status?
https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status

Reply:
I've added the status, thanks.

Reply:
seems good to go now, thanks for all work @lyctw

Reply:
Thank @kraj and @alistair23 for helping me.